test/test-run-express-metrics.js

```diff
@@ -0,0 +1,35 @@
+var helper = require('./helper');
```

Comment:
Strangely enough, the test is not working on my machine today, I'll investigate.